google/cloud/documentai_toolbox/wrappers/page.py

```diff
@@ -90,97 +106,184 @@ def sample_table_to_csv():
         return self.to_dataframe().to_csv(index=False)
 
 
-def _table_wrapper_from_documentai_table(
-    documentai_table: documentai.Document.Page.Table, text: str
-) -> Table:
-    r"""Returns a Table.
+@dataclasses.dataclass
+class Token:
+    """Represents a wrapped documentai.Document.Page.Token.
 
-    Args:
-        documentai_table (documentai.Document.Page.Table):
-            Required. A documentai.Document.Page.Table.
+    Attributes:
+        documentai_object (google.cloud.documentai.Document.Page.Token):
+            Required. The original google.cloud.documentai.Document.Page.Token object.
         text (str):
-            Required. UTF-8 encoded text in reading order
-            from the document.
+            Required. UTF-8 encoded text.
+    """
 
-    Returns:
-        Table:
-            A Table.
+    documentai_object: documentai.Document.Page.Token
+    text: str
 
+
+@dataclasses.dataclass
+class Line:
+    """Represents a wrapped documentai.Document.Page.Line.
+
+    Attributes:
+        documentai_object (google.cloud.documentai.Document.Page.Line):
+            Required. The original google.cloud.documentai.Document.Page.Line object.
+        text (str):
+            Required. UTF-8 encoded text.
     """
 
-    header_rows = _table_rows_from_documentai_table_rows(
-        table_rows=list(documentai_table.header_rows), text=text
-    )
-    body_rows = _table_rows_from_documentai_table_rows(
-        table_rows=list(documentai_table.body_rows), text=text
+    documentai_object: documentai.Document.Page.Line = dataclasses.field(
+        init=True, repr=False
     )
+    text: str = dataclasses.field(init=True, repr=False)
+    _page: Page = dataclasses.field(init=True, repr=False)
```

Comment:
Here the annotation can be: (note the quotes)
```
_page: "Page" = ...
```

Reply:
Also by the way I believe `init=True` is the default, so it does not need to be explicitly passed to `dataclasses.field`.